File: src/app/wyprawa/src/app/model/place.ts
export class Place {
  private _id: number;
  private _name: string;
  private _latitude: number;
  private _longitude: number;
  private _visited: boolean;
  private _description: string;

  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get name(): string {
    return this._name;
  }

  set name(value: string) {
    this._name = value;
  }

  get latitude(): number {
    return this._latitude;
  }

  set latitude(value: number) {
    this._latitude = value;
  }

  get longitude(): number {
    return this._longitude;
  }

  set longitude(value: number) {
    this._longitude = value;
  }

  get visited(): boolean {
    return this._visited;
  }

  set visited(value: boolean) {
    this._visited = value;
  }

  get description(): string {
    return this._description;
  }

  set description(value: string) {
    this._description = value;
  }
}

File: src/hakaton/Models/Place.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace hakaton.Models
{
    public class Place
    {
        public int ID { get; set; }
        public string Name { get; set; }
        public double Lenght { get; set; }
        public double Width { get; set; }
        public bool Visited { get; set; }
        public string Description { get; set; }


    }
}
